lib/canada_post/credentials.rb

```diff
@@ -9,7 +9,7 @@ def initialize(options={})
       @username         = options[:username]
       @password         = options[:password]
       @customer_number  = options[:customer_number]
-      @mode             = options[:mode] || "development"
+      @mode             = options[:mode] || 'development'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.